data_model/src/events/data/events.rs

```diff
@@ -346,17 +361,35 @@ mod domain {
         Deserialize,
         Serialize,
         IntoSchema,
-        Filter,
     )]
-    #[non_exhaustive]
     #[allow(missing_docs)]
+    pub struct AccountRoleChanged {
+        pub account_id: AccountId,
+        pub role_id: RoleId,
+    }
+}
+
+mod domain {
+    //! This module contains `DomainEvent` and its impls
+
+    use super::*;
+
+    type DomainMetadataChanged = MetadataChanged<DomainId>;
+
+    /// Domain Event
+    #[derive(
+        Clone, PartialEq, Eq, Debug, Decode, Encode, Deserialize, Serialize, IntoSchema, Filter,
+    )]
+    #[non_exhaustive]
+    // TODO: fix variant size differences
+    #[allow(missing_docs, variant_size_differences)]
```

Comment:
what about size differences?

Reply:
@mversic what do you mean?

Reply:
I think, he means that we should fix the variant size differences before merging.

Reply:
Fixed